internal/service/domain/sensor/mqtt.go

```diff
@@ -92,7 +92,7 @@ func (s *SensorService) updateSensorCoordsAndStatus(ctx context.Context, payload
 		if err != nil {
 			return nil, err
 		}
-
+		log.Debug("Coordinates and status of sensor have been updated successfully", "sensor_id", updatedSensor.ID)
```

Comment:
Maybe this should be an Info logging. Also, please be consistent in the log messages by writing everything in lower case